README.rst

```diff
@@ -447,6 +447,19 @@ of the url, this will enable you to point all the API urls to
 your openwisp-network-topology API server's domain,
 example value: ``https://mytopology.myapp.com``.
 
+``OPENWISP_NETWORK_TOPOLOGY_API_AUTH_REQUIRED``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++--------------+---------------+
+| **type**:    |   ``boolean`` |
++--------------+---------------+
+| **default**: |   ``False``   |
```

Comment:
let's default it to ``True`` because is a lot more common.

Reply:
I was thinking of introducing this as `False` and allowing users to which to True on their own. as setting it to `True` might break their work as alot of endpoints request which were working prior to this will now fail.

Reply:
Ok. Let me make this change